config/initializers/devise.rb

```diff
@@ -0,0 +1,299 @@
+# frozen_string_literal: true
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
```

Comment:
@gallosanchez23 we should remove as many of these comments as we can before merging. Let's only leave the absolutely essential ones (hopefully none) :).